automation-generators/generic/cp4wa/preprocessor.py

```diff
@@ -0,0 +1,302 @@
+from generatorPreProcessor import GeneratorPreProcessor
```

Comment:
This preprocessor  pertains to cp4d. If you don't have a preprocessor yet, pease remove the .py file and no linting will take place.

Reply:
That preprocessor expands the OCS storage to the actual StorageClasses. The script fails if I remove it.

Reply:
I can look at the file and try and strip out some of the unnecessary stuff if you'd like.